static/app/components/modals/inviteMissingMembersModal/index.tsx

```diff
@@ -0,0 +1,332 @@
+      {
+        organization,
+      },
+      {startSession: true}
+    );
+  };
+
+  const selectedCount = memberInvites.filter(i => i.selected).length;
+  const selectedAll = memberInvites.length === selectedCount;
+
+  const inviteButtonLabel = () => {
+    return tct('Invite [memberCount] missing member[isPlural]', {
+      memberCount:
+        memberInvites.length === selectedCount
+          ? `all ${selectedCount}`
+          : selectedCount === 0
+          ? ''
+          : selectedCount,
+      isPlural: selectedCount !== 1 ? 's' : '',
+    });
+  };
+
+  const hookRenderer: InviteModalRenderFunc = ({sendInvites, canSend, headerInfo}) => (
+    <Fragment>
+      <h4>{t('Invite Your Dev Team')}</h4>
+      {headerInfo}
+      <StyledPanelTable
+        headers={[
+          <Checkbox
+            key={0}
+            aria-label={selectedAll ? 'Deselect All' : 'Select All'}
+            onChange={() => selectAll(!selectedAll)}
+            checked={selectedAll}
+          />,
+          t('User Information'),
+          <StyledHeader key={1}>
+            {t('Recent Commits')}{' '}
+            <Tooltip title={t('Based on the last 30 days of commit data')}>
+              <IconInfo size="xs" />
+            </Tooltip>
+          </StyledHeader>,
+          t('Role'),
+          t('Team'),
+        ]}
+      >
+        {memberInvites?.map((member, i) => {
+          const checked = memberInvites[i].selected;
+          return (
+            <Fragment key={i}>
+              <div>
+                <Checkbox
+                  aria-label={t('Select %s', member.email)}
+                  checked={checked}
+                  onChange={() => checkboxToggle(!checked, i)}
+                />
+              </div>
+              <StyledPanelItem>
+                <ContentRow>
+                  <IconGithub size="sm" />
+                  <StyledExternalLink href={`http://github.com/${member.externalId}`}>
+                    {tct('@[userId]', {userId: member.externalId})}
+                  </StyledExternalLink>
+                </ContentRow>
+                <Subtitle>{member.email}</Subtitle>
+              </StyledPanelItem>
+              <ContentRow>
+                <IconCommit size="sm" />
+                {member.commitCount}
+              </ContentRow>
+              <RoleSelectControl
+                aria-label={t('Role')}
+                data-test-id="select-role"
+                disabled={false}
+                roles={invitableRoles}
+                disableUnallowed
+                onChange={value => setRole(value?.value, i)}
+              />
+              <TeamSelector
+                aria-label={t('Add to Team')}
+                data-test-id="select-teams"
+                disabled={false}
+                placeholder={t('Add to teams\u2026')}
+                onChange={opts => setTeams(opts ? opts.map(v => v.value) : [], i)}
+                multiple
+                clearable
+              />
```

Comment:
team selector expects org to be in context which it isn't in modals. you'll probably run into the same thing as this PR is trying to fix https://github.com/getsentry/sentry/pull/55029

Reply:
You should be able to pass organization down since its using withOrganization